input/fsh/DkCoreObservation.fsh

```diff
@@ -46,7 +46,7 @@ Description: "HL7 Denmark core profile for observations"
   * code 1..
 * subject only Reference(DkCorePatient or Group or Location or Device)
 * subject 1..
-* performer only Reference(DkCorePractitioner or DkCoreOrganization or DkCorePatient or PractitionerRole or CareTeam or RelatedPerson)
+* performer only Reference(DkCorePractitioner or DkCoreOrganization or DkCorePatient or PractitionerRole or CareTeam or DkCoreRelatedPerson)
```

Comment:
This change is not reflected in the description under "Subject and Performers" where RelatedPerson should be a link like the 3 other profiled resource types.

Reply:
Link tilføjes. @christiangasser-lakeside